### ISSUE TITLE
Update Tooling API to API 32.0

### ISSUE THREAD

**Issue by afawcett** [afawcett/apex-toolingapi#24]
Specific ask [here](http://salesforce.stackexchange.com/questions/57526/are-there-any-plans-to-update-toolingapi-cls-for-apex-to-v32-in-order-to-support) to support AuraDefinitionBundle.

**Comment by dancinllama** [comment]
You mean 31? :)

**Comment by brandonmikeska** [comment]
Any way I can update the code to work with the latest API 41.0?

**Comment by afawcett** [comment]
Yes sure just change the version number in the .xml and package.xml file. If you want to do a PR that would be great.

**Comment by afawcett** [comment]
Ok, so this is a handwritten wrapper (unlike the MD SOAP API wrapper). So you would need to add the types yourself and update the endpoint used.

**Comment by brandonmikeska** [comment]
Would it make sense to generate the SOAP Class that Salesforce has and copy and past the newly added types?

There shouldn’t be too many types, but don’t know how else you would validate the types are correct. Any guidance would be great, or I can take a first stab and you can review the pull request. Whatever works best.

**Comment by afawcett** [comment]
The problem with the tooling API WSDL is that it has very heavy dependency on inhirtance in its types and runtime use of the xsi:type attribute to denote the instance type of the XML fragement. The generated code from the WSDL2Apex generator from Apex does not understand this and further more the runtime marhsalling/unmarshalling engine does not. 

Here is what i think should be done to this project
1. Move the master branch back to the REST API version 
2. Update the REST API classes manually based on what we see in the docs and WSDL. Also perhaps provide support in the base type used for unmapped JSON fields. 

What do yuo think?
